lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1545,3 +1545,57 @@ CompilerInvocation::loadFromSerializedAST(StringRef data) {
                         extendedInfo.getExtraClangImporterOptions().end());
   return info.status;
 }
+
+bool CompilerInvocation::getOrCreatePCH() {
+
+  const auto &BridgingHeader = getClangImporterOptions().BridgingHeader;
+
+  const auto &PCHOutputDir = getClangImporterOptions().PrecompiledHeaderOutputDir;
+
+  if (BridgingHeader.empty() || PCHOutputDir.empty()) {
+    return true;
+  }
+
+  CompilerInstance ReadInstance;
+  auto FailedToGetReadInstance = ReadInstance.setup(*this);
```

Comment:
Please do not create an entire separate Swift CompilerInstance just to make a Clang importer and have it do one thing.